### DISCUSSION TITLE
Double scans when using clamav

### DISCUSSION THREAD

**Discussion by DanjelJungersen** [rspamd/rspamd#5130] [Q&A]
Hi there.

I have set up clamav, and I think it works....
But when a mail is recieved, it is first scanned by rspamd and the clamav. Thats all fine.
But when clamav is done, rspamd scans it again.

My setup is debian, postfix, rspamd, clamav, dovecot.
Most of the setup from https://workaround.org/ispmail-bookworm/

This causes 2 concerns for me.
1) The extra work done by the server (Not very busy, so the a very big issue)
2) The second scan has a higher score, so I fear that the second scan goes above the reject setting and the server rejects mail from itself :-(

How to avoid this?

My first guess is to disable rspamd for mails coming from clamav, so that the second scan never happens.
This will solve both my issues.
But I have not found a solution, though I have REALLY tried googling.

Anyone?
Danjel

**Comment by fatalbanana** [comment]
Maybe [metadata_exporter module](https://rspamd.com/doc/modules/metadata_exporter.html) is configured to do something here, or it's a story about MTA configuration.

[Settings](https://rspamd.com/doc/configuration/settings.html) module might be useful to suppress scanning.
